XRTK.SDK/Packages/com.xrtk.sdk/Runtime/Features/UX/Scripts/Controllers/Hands/HandControllerJointsVisualizer.cs

```diff
@@ -4,17 +4,17 @@
 using System.Collections.Generic;
 using UnityEngine;
 using XRTK.Definitions.Controllers.Hands;
-using XRTK.EventDatum.Input;
 
 namespace XRTK.SDK.UX.Controllers.Hands
 {
-    /// <summary>
-    /// Hand controller visualizer visualizing hand joints.
-    /// </summary>
-    [System.Runtime.InteropServices.Guid("0B653537-D237-4622-ACC6-E351209A5882")]
-    public class HandControllerJointsVisualizer : BaseHandControllerVisualizer
+    public class HandControllerJointsVisualizer : MonoBehaviour
```

Comment:
We probably still need the guid, no?

Reply:
I don't think. This is now just a plain MonoBehaviour component in the SDK. It's not being assigned in the controller visualization profiles.